auth-server/src/main/java/com/antonio/authserver/service/JwtService.java

```diff
@@ -67,7 +69,7 @@ public Long getRefreshTokenExpirationTime() {
 
     private String buildToken(String issuer, String subject, long expirationTime,
                               Collection<? extends GrantedAuthority> authorities, String secretKey) {
-        String token = Jwts.builder().setIssuer(issuer).setSubject(subject).claim("authorities", authorities)
+        String token = Jwts.builder().setIssuer(issuer).setSubject(subject).claim("authorities", authorities).setIssuedAt(Calendar.getInstance().getTime())
```

Comment:
pentru sesiuni ai facut setarea asta?

Reply:
@ToniVinter

Reply:
Dada, am adaugat data la care se creeaza tokenul

Reply:
@ToniVinter dar la ce folosesti? pentru ca in token aveai deja expiration time, care ti-ar spune daca e sau nu valid token-ul